abkfenris.md

```diff
@@ -0,0 +1,5 @@
+# Alex Kerney
+
+I'm a web application developer at the Gulf of Maine Research Institute from Boothbay Harbor, ME.
+
+Giffords strawberry ice cream is pretty dang good. There are also a lot of fantastic black raspberry options out there.
```

Comment:
Could add some clarity on why it is "pretty dang good" for those seeking to recreate such a dairy product at home